### ISSUE TITLE
crash while parsing version contain hash

### ISSUE THREAD

**Issue by trint30** [timvisee/version-compare#26]
Hi,

I have some trouble while parsing some non-standard version like the last one :
```
fn test_version() {
	use version_compare::{Cmp, Version};
	println!("Ok {:?}", Version::from("0.0.1-test.0222426166a"));
	println!("Ok {:?}", Version::from("0.0.1-test.0222426166565421816516584651684351354"));
	println!("Crash {:?}", Version::from("0.0.1-test.02224261665a"));
	println!("Crash {:?}", Version::from("0.0.1-test.02224261665d7b1b689816d12f6bcacb"));
}
```

Output : 
```
thread 'utils::test_version' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: PosOverflow }', ~/.cargo/registry/src/github.com-1ecc6299db9ec823/version-compare-0.1.0/src/version.rs:347:65
stack backtrace:
   0: rust_begin_unwind
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/core/src/panicking.rs:142:14
   2: core::result::unwrap_failed
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/core/src/result.rs:1814:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/core/src/result.rs:1107:23
   4: version_compare::version::split_version_str
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/version-compare-0.1.0/src/version.rs:347:45
   5: version_compare::version::Version::from
             at ~/.cargo/registry/src/github.com-1ecc6299db9ec823/version-compare-0.1.0/src/version.rs:49:20
```
Is it possible to handle that instead of panic ?

**Comment by timvisee** [comment]
Woah! I never expected this to panic. Thanks for your report.

No, I'm afraid this can't be handled right now. I'll try to release a fix as soon a possible. Feel free to submit a PR in the meanwhile.

**Comment by timvisee** [comment]
I've fixed this panic in `0.1.1` which I've just released. Please update and see if the issue persists.

The issue wasn't caused by parsing a hash per se, but because it was trying to fit an arbitrary long number in the `i32`. When that didn't fit, it paniced. This doesn't happen anymore, instead it handles these large numbers as text.

Thanks once again for the report!

**Comment by trint30** [comment]
Hi, what a quickness !  :)

I just update the new version `0.1.1`. It works perfectly. Thank you !
